client/src/components/game/lab2/LandingPage/landingPage.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Button from "../components/header/buttons/button";
+import MainInstructions from "./mainInstructions";
+
+const LandingPage = ({
+  endFirstGame,
+  toWhiteBackground,
+  background,
+  startGame
+}) => {
+  const closePage = () => {
+    endFirstGame();
+  };
+
+  if (background !== "white") {
+    toWhiteBackground();
+  }
+
+  return (
+    <div>
+      <div id="Header">
+        <p className="mainTitle">Welcome!</p>
+      </div>
+      <div id="Body">
+        <MainInstructions />
+          <div className="mainInstructionsContainer">
+            <p
+              className="mainInstructionList"
+              style={{
+                marginTop: "40px",
+                marginBottom: "10px",
+                textAlign: "left"
+              }}
+            >
+              To help track your game history and to help ensure we are
```

Comment:
The line spacing of this blurb is different than the other instructions on the beginning page
<img width="855" alt="Screen Shot 2021-03-08 at 5 34 41 PM" src="https://user-images.githubusercontent.com/54957497/110390855-9c11d080-8034-11eb-8a48-1d64e03c1a0f.png">